### ISSUE TITLE
Version 6.2.0 not released for scala 2.12

### ISSUE THREAD

**Issue by Philippus** [gzoller/ScalaJack#199]
I think version 6.2.0 was only released for scala 2.13. Was this intentional?

**Comment by gzoller** [comment]
Hi, Philippus... Sorry for the long delay!  I've been busy working on the Dotty/Scala 3 port of ScalaJack, which was very consuming.  The Scala people removed runtime reflection entirely!  I had to engineer a way to bring it back.  

How important is this to you?  There are quite a few structural differences between 2.12 and 2.13, and in December 2019 I released code that cleaned up a lot and was written in a 2.13-specific style and removed the 2.12 support.

It's possible to re-enable 2.12, but its a fair bit of work, so I wanted to double-check with you first to see if this was nice-to-have or mission-critical for you.
